packages/app/cypress.config.ts

```diff
@@ -1,7 +1,7 @@
 import { defineConfig } from 'cypress'
 
 export default defineConfig({
-  'projectId': 'sehy69',
+  'projectId': process.env.CYPRESS_INTERNAL_ENV === 'production' ? 'sehy69' : 'ypt4pf',
```

Comment:
I think this should be something like `process.env.CYPRESS_INTERNAL_ENV === 'staging' ? 'ypt4pf' : 'sehy69',` because the tests are pointing to staging but the project slug and the record key do not match, making those fail 
https://app.circleci.com/pipelines/github/cypress-io/cypress/27699/workflows/6f949225-c3a5-46b9-bfea-62399b8ce41f/jobs/1049202